examples/controlplane/controlplane.yaml

```diff
@@ -43,3 +43,87 @@ spec:
     controllerManager:
       extraArgs:
         cloud-provider: aws
+---
+apiVersion: cluster.x-k8s.io/v1alpha2
+kind: Machine
+metadata:
+  name: ${CLUSTER_NAME}-controlplane-1
+  labels:
+    cluster.x-k8s.io/control-plane: "true"
+    cluster.x-k8s.io/cluster-name: "${CLUSTER_NAME}"
+spec:
+  version: ${KUBERNETES_VERSION}
+  bootstrap:
+    configRef:
+      apiVersion: bootstrap.cluster.x-k8s.io/v1alpha2
+      kind: KubeadmConfig
+      name: ${CLUSTER_NAME}-controlplane-1
+  infrastructureRef:
+    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+    kind: AWSMachine
+    name: ${CLUSTER_NAME}-controlplane-1
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+kind: AWSMachine
+metadata:
+  name: ${CLUSTER_NAME}-controlplane-1
+spec:
+  instanceType: ${CONTROL_PLANE_MACHINE_TYPE}
+  iamInstanceProfile: "control-plane.cluster-api-provider-aws.sigs.k8s.io"
```

Comment:
To support pivoting, all of these will need to be updated with the `controllers.cluster-api-provider-aws.sigs.k8s.io` role